openqabot/approver.py

```diff
@@ -123,13 +123,22 @@ def osc_approve(inc: IncReq) -> bool:
             )
         except HTTPError as e:
             if e.code == 403:
-                logger.debug(
+                logger.info(
                     "Received '%s'. Request likely already approved, ignoring"
                     % e.reason
                 )
                 return True
+            elif e.code == 404:
+                logger.error(
```

Comment:
still here I would use `logger.debug` (or `logger.info`) as there is already the action mentioned that is conducted when the situation is hit. We should ask ourselves what a reader of an error log message should do. If it's nothing that needs to be done then we also don't need to log with that high level